packages/hooks/src/useWithdrawLocked.ts

```diff
@@ -0,0 +1,17 @@
+import { useStore } from '@deriv/stores';
+import useNeedPOI from './useNeedPOI';
+
+const useWithdrawLocked = () => {
+    const { client } = useStore();
+    const is_need_poi = useNeedPOI();
+
+    if (!client.account_status?.status) {
```

Comment:
@hamid-deriv TS says none of them are nullable 🤔 I guess the types are wrong then, Right? Should we fix the type? 🤔

Reply:
The `account_status` can be empty. This is for checking if the `status` is provided inside the `account_status`.

Reply:
@hamid-deriv That's what I mean, Shouldn't we update the type somehow to get errors for this? 🤔

Reply:
Sorted